### ISSUE TITLE
Marks not attaching in sans

### ISSUE THREAD

**Issue by simoncozens** [notofonts/oriya#13]
As noted in https://github.com/google/fonts/pull/4987#issuecomment-1206548565, there are some bad mark attachments in Sans, particularly with the following sequences:

ରେଅ୍ଥ
ନ୍ମ ? 
ପ୍ତୁଂ ଅ

**Comment by simoncozens** [comment]
OK, for posterity: I fixed the presenting issue here - these specific mark/base combinations. However, there do seem to be a large number of compound bases which don't have any anchors, so there may be many more attachment issues out there. *However however*, conjuncts on these bases don't seem to occur in real Odia text (I just downloaded the whole of Odia wikipedia and checked...). It's only things like Sanskrit written with Oriya that seems to make use of it.

So there may be other problems here, but they equally may not really matter. I'm not sure.
